spec/support/flores_extensions.rb

```diff
@@ -1,6 +1,46 @@
 # encoding: utf-8
 require "flores/pki"
+require "flores/random"
+
 module Flores
```

Comment:
This is an awesome util! I assume this was done as a monkey patch due to @jordansissel being out of pocket for a while?

Have you considered just putting this in some other namespace for now? Monkey patching is kind of dicey. Ideally this would just be in `BeatsInputTestHelpers` or the like, and we'd open a PR agains Flores to add this in, then swap it out if that was accepted.